playwright/api/call-blockfrost/blockfrost.api.ts

```diff
@@ -24,7 +24,37 @@ export function blockfrostApi(request: APIRequestContext) {
   const getLastEpochData = async () => {
     return BaseApi.getData(
       request,
-      Endpoint.BlockFrost.LastEpochData.Base,
+      Endpoint.BlockFrost.EPOCHS.Base + "/latest",
```

Comment:
/latest should be in endpoints.ts - why? becasue you want central place to handle endpoints as per comment in endpints.ts